lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -148,11 +167,22 @@ window.runLighthouseInExtension = function(options, requestedAggregations) {
   // Default to 'info' logging level.
   log.setLevel('info');
   const connection = new ExtensionProtocol();
-  return connection.getCurrentTabURL()
+  return enableOtherChromeExtensions(false)
+    .then(_ => connection.getCurrentTabURL())
     .then(url => window.runLighthouseForConnection(connection, url, options, requestedAggregations))
     .then(results => {
+      // Deliver results even if enabling extensions fails.
+      enableOtherChromeExtensions(true).catch(err => {
+        log.warn('Chrome', `Could not enable some extensions. ${err.message}`);
```

Comment:
I think I remember it throwing, but will check. If not, we can monitor `lastError`.